docs/protocol-design/distribution-and-units.md

```diff
@@ -34,7 +34,7 @@ A 128 bit integer is used to represent account balances.  A set of SI prefixes[^
 |               | μnano/unano | 1000000000000000000                | $10^{18}$
 | raw           |             | 1                                  | $10^{0}$
 
-1 raw is the smallest possible division and NANO/Nano (Mnano) is the current standard division used in most wallets, on exchanges, etc.
+1 raw is the smallest possible division and NANO/Nano (Mnano) is the current standard division used in most wallets, on exchanges, etc.  In other words, NANO is the ticker symbol on exchanges, representing 1 million nano. All block-related [RPC commands](/commands/rpc-protocol/) only accept `raw` amounts, and most [RPC commands](/commands/rpc-protocol/) accept amounts as decimal integers for convenience.  Many wallet implementations only make receive-blocks for amounts bigger than 1 nano (i.e. $10^{24}$ raw) by default.
```

Comment:
```suggestion
1 raw is the smallest possible division and NANO/Nano (Mnano) is the current standard division used in most wallets, on exchanges, etc. In other words, NANO is the ticker symbol on exchanges, representing 1 million nano. All block-related [RPC commands](/commands/rpc-protocol/) only accept `raw` amounts, and amounts are always given in requests and responses as decimal strings. The node wallet only receives amounts larger than 1 nano (i.e. $10^{24}$ raw) by default.
```

What do you think of this change? RPCs accept amounts as strings, not decimal integers, to avoid issues with the built-in json conversion in some languages which could not support 128-bit integers.